core/src/main/resources/error/error-classes.json

```diff
@@ -1432,6 +1432,12 @@
     },
     "sqlState" : "0A000"
   },
+  "UNSUPPORTED_EXPR_FOR_OPERATOR" : {
+    "message" : [
+      "The query operator `<operator>` contains one or more unsupported expression types Aggregate, Window or Generate.",
```

Comment:
We shouldn't show users internal things like Generate. Could you reformulate  this, for example https://github.com/apache/spark/blob/04d72659a88c3e94ff97f27e5481bc130b824b0a/core/src/main/resources/error/error-classes.json#L939

Reply:
Thanks! Just applied the comment